mmv1/third_party/terraform/tests/resource_vmwareengine_cluster_test.go.erb

```diff
@@ -0,0 +1,179 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+	"strings"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccVmwareengineCluster_vmwareEngineClusterUpdate(t *testing.T) {
```

Comment:
This is failing to finish creating the private cloud within 3 hours - is there any chance there's a server-side issue here, or do we need an even longer timeout?

Error:

```
Error: Error waiting to create PrivateCloud: Error waiting for Creating PrivateCloud: timeout while waiting for state to become 'done: true' (last state: 'done: false', timeout: 3h0m0s)
```

Reply:
yes I have merged the expand and shrink tests. This would ideally take more than 3 hours, because we have to provision additional 4 node cluster and expand and shrink.
Is there any way to specify timeouts at test level for tests I think currently its overall test runner level single timeout.

Reply:
Is there a way to find the logs for the requests. I can't see any private cloud currently in the region for the tests. Is it automatically cleaned up. Can you please check the CI project to confirm once.

Reply:
from discussion - this was a case where the cloud was created eventually. may need to increase timeout.